docs/manifest/sourcelocation.md

```diff
@@ -1,13 +1,15 @@
 ---
 title: SourceLocation element in the manifest file
 description: The SourceLocation element specifies the source file locations for your Office Add-in.
-ms.date: 06/14/2022
+ms.date: 12/01/2023
 ms.localizationpriority: medium
 ---
 
 # SourceLocation element
 
-Specifies the source file locations for your Office Add-in as a URL between 1 and 2018 characters long. The source location must be an HTTPS address, not a file path.
+Specifies the source file locations for your Office Add-in as a URL between 1 and 2018 characters long. The source location must be an HTTPS address, not a file path. 
+
+The source location can be any website, and you can use any directory structure you want. Typically, all the files in the add-in, except the manifest, are deployed to the source location. In the case of a single-page application, there is usually just one file. You can host some files on a CDN, or other domain, provided that you register the domain with an [AppDomain](appdomain.md) element. You would include a copy of the manifest at your source location only if your add-in needs to read a manifest value at runtime. 
```

Comment:
Just a couple suggestions to make the text shorter and flow a little better (IMO).
```suggestion
The source location can be any website, and you can use any directory structure you want. Typically, all the files in the add-in, except the manifest, are deployed to the source location. For a single-page application, there is usually one file. You can host files on a CDN, or other domain, provided that you register the domain with an [AppDomain](appdomain.md) element. You would only include a copy of the manifest at your source location if your add-in needs to read a manifest value at runtime. 
```

Reply:
> Just a couple suggestions to make the text shorter and flow a little better (IMO).

Now that I think about it, there is never a good reason to put the manifest with the other files, so I just removed the last sentence. I modified your other suggestion so as not to imply that *all* the files can be at a CDN. There has to be at least one at the source location.